docs/make-all-versions.sh

```
@@ -0,0 +1,108 @@
+    cat "${TMPDIR}/"/conf.py.versions >> docs/source/conf.py
+
+    make clean
+    make sync
+    make docs
+
+    git stash # un-do uv.lock
+    git checkout HEAD
+    # remove the front v from the release tag
+    release=$(echo $release | sed 's/^v//')
+
+    mv docs/build/html docs/build_all/$(echo $release | tr '[:upper:]' '[:lower:]')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)